### ISSUE TITLE
def statement gets deleted

### ISSUE THREAD

**Issue by tgy** [dadadel/pyment#32]
Running pyment (with fix https://github.com/dadadel/pyment/pull/31) on this file
```python
def hello_world(a=22, b='hello'):
  return 42
```

Gives the following patch
```diff
# Patch generated by Pyment v0.3.2

--- a/test.py
+++ b/test.py
@@ -1,2 +1,15 @@
-def hello_world(a=22, b='hello'):
+    '''
+
+    Parameters
+    ----------
+    a :
+         (Default value = 22)
+    b :
+         (Default value = 'hello')
+
+    Returns
+    -------
+
+    
+    '''
   return 42
```

It deletes the first line.

Now I may be wrong but I'm pretty sure this is not the expected output! 😉

**Comment by dadadel** [comment]
Thanks!
But this issue is not related to any fix.
It happens when a file starts with a signature of a function or a class.

**Comment by tgy** [comment]
I was just mentioning that I was using fix #31 because otherwise the code fails (with `numpydoc`) so it is not reproducible as is.

**Comment by dadadel** [comment]
I have fixed the issue #32 and added a test for it. I close it. If there's still a problem don't hesitate to reopen it.
Thanks for your help @tgy!
